be/src/service/user.dto.ts

```diff
@@ -0,0 +1,6 @@
+export interface UserDTO {
+    name: string;
+    password: string;
+    email: string;
+  }
+  
```

Comment:
For some reason github is not happy with the line ending here 
![image](https://user-images.githubusercontent.com/33936483/116058325-64b0bf00-a688-11eb-93f9-52295c81d8bd.png)

Reply:
yeah forgot to run lint